kolibri/plugins/coach/assets/src/views/plan/GroupMembersPage/index.vue

```diff
@@ -22,11 +22,12 @@
         </h1>
 
         <KFixedGrid numCols="2">
-          <KFixedGridItem
-            span="1"
-            class="number-learners"
-          >
-            {{ coachString('numberOfLearners', { value: currentGroup.users.length }) }}
+          <KFixedGridItem span="1">
+            <KIcon
+              icon="classes"
+              class="class-name-icon"
+            />
```

Comment:
I did not use a KLabeledIcon here because the classes icon is, by default, far too small compared to the text.